pyproject.toml

```diff
@@ -58,7 +58,6 @@ module = [
     "django_ftl",
     "django_ftl.bundles",
     "google.*",
-    "google_measurement_protocol",
```

Comment:
This is the only line I meant to change in here. The rest were done by my editor's TOML formatter.